ocaml/xapi/test_host.ml

```diff
@@ -0,0 +1,84 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Stdext
+open OUnit
+open Test_common
+
+module D=Debug.Make(struct let name="test_xapi_xenops" end)
+open D
+
+let add_host __context name =
+  ignore(Xapi_host.create ~__context
```

Comment:
It's not important, but I think we could use `Test_common.make_host ~__context ~name_label:name ()` here

Reply:
True, but given this is function I actually want to test, I thought I'd call it explicitly.

Reply:
Good idea, I agree.